client/src/components/LoadingWheel/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from './LoadingWheel.module.css';
+
+const LoadingWheel = () => (
+  <div id={styles['loader-container']}>
+    <div className={styles.loader} />
+  </div>
+);
```

Comment:
bit late now but there are some lovely loading wheels part of Material UI: https://material-ui.com/components/progress/

Reply:
Good find - can be added in future and made as issue #52